src/constants/leadStatuses.ts

```diff
@@ -0,0 +1,11 @@
+export const leadStatuses = {
+  NEW: { label: 'New', id: 'b041e847-a905-47c1-9f79-430e2fbd6db6' },
+  ASSIGNED: { label: 'Assigned', id: '97e6703e-6282-41d8-91d9-8a911ed6b321' },
+  CONTACT_MADE: { label: 'Contact made', id: '3f6a4fc9-1378-4fca-ad91-894090a6dc72' },
+  VIEWING: { label: 'Viewing', id: '58807212-5e4a-424e-8f69-ece771f03f01' },
+  PROPOSAL: { label: 'Proposal', id: '36760639-bf6d-4dcd-8d9d-25fab4f39254' },
+  NEGOTIATION: { label: 'Negotiation', id: '58ad135b-5036-415b-8ef2-38b417e8ffb4' },
+  ASLEEP: { label: 'Asleep', id: '64622f10-f4ba-4aa1-afb0-73b2571ccc77' },
+  CLOSED_WON: { label: 'Closed won', id: '6ec12111-6ed8-4bed-ac80-4e49e9cefb8b' },
+  CLOSED_LOST: { label: 'Closed lost', id: 'a7ebcea2-975a-4b34-93e7-59eb161c53bc' },
+};

```

Comment:
why did you hardcode it ?

Cant you just get a list from api and transform it into select options ?

Reply:
Unfortunately, there is no way to get the list from the api

Reply:
https://trial5251.eu1.qobrix.com/api/v2/workflows/72c670a3-0521-4b6f-8ce8-6c8299e1dad0/stages

Reply:
its not an option at all, when we will change the API key, all will failed as new CRM will have own ip

Reply:
Lyudmyla, it's not api id, it's a crm statuses ids. They won't change

Reply:
I'm not talking about users. I mean when we change CRM API key, they will create other instance of db for us = other db = other id